src/containers/Chat/ChatMessages/AddVariables/AddVariables.tsx

```diff
@@ -95,9 +95,10 @@ export const AddVariables: React.FC<AddVariablesPropTypes> = ({
             alignButtons="left"
           >
             <div data-testid="variablesDialog">
-              {formFieldItems.map((field: any) => (
-                <Field {...field} />
-              ))}
+              {formFieldItems.map((field: any, index: any) => {
+                const key = index;
```

Comment:
We should not use index as keys. Can you check if we have something unique like a name that can be added.